x-pack/plugins/index_management/public/sections/index_list/components/index_table/index_table.js

```diff
@@ -102,6 +107,9 @@ export class IndexTableUi extends Component {
       filterChanged(EuiSearchBar.Query.parse(decodedFilter));
     }
   }
+  componentDidUnmount() {
```

Comment:
I think this needs to be changed to `componentWillUnmount`. Here's the React warning in the browser console:

![image](https://user-images.githubusercontent.com/1238659/49890750-bd55c900-fdf9-11e8-9be8-febf74971606.png)

Reply:
fixed.